### PR TITLE
About

### PR DESCRIPTION
**Pull request by mjurfest**
#### Trello board reference:

* [Trello Card](https://trello.com/c/pm8uCZ4P/20-2-as-a-user-i-should-be-able-to-check-the-about-section-so-i-can-learn-more-about-target-mvd)

---

#### Description:

* Check About section of the app in the home page

---

### CODE REVIEW

File: spec/requests/api/v1/settings/show_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe 'GET api/v1/settings/show', type: :request do
+  let(:setting) { create(:setting) }
+
+  it 'returns success' do
+    get api_v1_settings_path(setting: 'About'), as: :json
```
Comment (santiagovidal) [inline spec/requests/api/v1/settings/show_spec.rb]: Keep in mind `About` is not a valid setting. Maybe use `setting.name` so it's clearer.

File: app/admin/setting.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
+ActiveAdmin.register Setting do
+# See permitted parameters documentation:
```
Comment (santiagovidal) [inline app/admin/setting.rb]: Delete this commented code.

File: config/routes.rb [hunk 3]
```diff
@@ -20,6 +20,7 @@
         end
         resources :targets, only: %i(create index show destroy)
         resources :contacts, only: :create
+        get :settings, to: 'settings#show'
```
Comment (fedeagripa) [inline config/routes.rb]: please use resources :settings, only: :show

File: spec/models/setting_spec.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+describe Setting do
+  describe 'validations' do
+    subject { build :setting }
+
+    it { should validate_presence_of(:name) }
+    it { should validate_presence_of(:info) }
```
Comment (Alxzu) [inline spec/models/setting_spec.rb]: What about validating the uniqueness of?

File: spec/requests/api/v1/settings/show_spec.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    expect(response).to have_http_status(:success)
+  end
+
+  it "returns setting data" do
```
Comment (Alxzu) [inline spec/requests/api/v1/settings/show_spec.rb]: Single quote

File: config/routes.rb (hunk 3 above)
Comment (Alxzu) [inline config/routes.rb]: What about `resources :settings, only: :show` as you did before?

**Pull request merged**